test/IntegrationTests/Helpers/TestHelper.cs

```diff
@@ -102,7 +102,7 @@ public void EnableDefaultExporters()
     /// RunTestApplication starts the test application, wait up to DefaultProcessTimeout.
     /// Assertion exceptions are thrown if it timed out or the exit code is non-zero.
     /// </summary>
-    public void RunTestApplication(TestSettings testSettings = null)
+    public (string StandardOutput, string ErrorOutput) RunTestApplication(TestSettings testSettings = null)
```

Comment:
Can we define a type for the result? I think it would be more readable. The type could be reused in `PowershellHelper`. In future we could add more fields e.g. `ExitCode`. WDYT?

Reply:
I hope YAGNI. It is completely internal API. It should be pretty easy to refactor it later.